src/main/java/com/movie/store/util/HibernateUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.movie.store.util;
+
+import com.movie.store.exceptions.DataProcessingException;
+import lombok.NoArgsConstructor;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+@NoArgsConstructor
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new DataProcessingException("Error creating session factory", e);
```

Comment:
```suggestion
            throw new RuntimeException("Error creating session factory", e);
```

this exception has a different nature